### DISCUSSION TITLE
Abandoning Wyze/Exsisting firmware?

### DISCUSSION THREAD

**Discussion by brokendash** [gtxaspec/wz_mini_hacks#209] [Development]
Hello,

Forgive me if this is a dumb qeustion, but why not dump the whole Wyze support entirely.... I do not care to use their app to configure and would preffer to configure the hardware using flat files on the SD card... ie: wifi etc..  The thing that I thought would be the imediate issue at some point when I first unboxed and setup the wyze cam was how easily it was for wyze to render it obsolete. I think this is a really neat project and thought it would be interesting to future proof the devices so they could be configured/setup etc.. even if Wyze goes belly up.....

Also would is be possible to run almost any brands of camera software that built on example: the Ingenix T31 SoC's? within reason?

Keep up the good work!
B

**Comment by endertable** [comment]
I believe the main purpose of this project is what many folks wanted:  something to supplement the Wyze eco-system but still be usable offline if need be and have more control of the cam.  Once you have configured the cams initially with Wyze, you can for the most part go offline from Wyze with some of the older firmwares if RTSP is all you need/want.
I have written a few times on here how to get a cam going straight out of the box (without the app) using this project, but that would be mostly useful for when you said:  when/if Wyze goes belly up (or you are off the internet grid).  Otherwise, setting up with the app is super easy (I have set up over 30 Wyze cams).
To completely get rid of Wyze one would need to write a whole new set of binaries to initialize the camera video hardware and some other stuff.  Although not impossible, is probably not within the scope of this project.  I have yet to see anyone do that with the V3 cams so I guess the cam manufacturer made some things a little more complicated from that aspect.

Things are always evolving so you never know...

**Comment by virmaior** [comment]
For me, I'm probably fine with using Wyze Fimrware +  wz_mini_hack _intranet only_ and then ssh, RTSP, and scp (to download from the SD cards), so I don't know if I need to go so far as a full wyze-independent app. **Moreover I don't presently have the right skills and don't have the time to learn them right now due to some life circumstances.** Even if Wyze starts getting aggressive with trying to update firmware etc., they can't do anything to cameras that are offline.

Towards that end, what I've been brewing is a web-based interface that shows me all of my cameras updated 1x/second. This would be trivial but my home network involves 2 routers and then 1 wifi-bridge for the main network and the cameras are one further subnet away through a raspberry pi (the routers are not configurable so I can't set a route very easily in them). Once I add RTSP to HLS for the feeds to my pi, then I've got something that would let my wife see all of the cameras whenever she wants, switching which one is live.  At that point, the only thing I'd still want/need is control for turning on/off IR lights and night modes.

That said I actually think there's some pedigree that would help simplify the task.
wz_mini borrows ideas and references atomcam_tools (https://github.com/mnakada/atomcam_tools)
atomcam_tools borrows ideas and references openmiko (https://github.com/openmiko/openmiko)

I think openmiko is a ground-up app for T20s.

**Reply by bjs-pdx** [reply]
For the V1/V2 and Pan V1, the Dafang firmware pretty much does what you ask...flat files and no Wyze.  No such firmware exists currently (as far as I know) for the other members for the Wyze camera family.

**Comment by gtxaspec** [comment]
openipc will probably be the project to keep an eye out for T31 devices.  I donated a V3 to them. 

They already have some wired T31 devices working, hopefully they can get the V3 working as well.  Check out their website www.openipc.org 

They still need a V2, PanV2 and Outdoor V2 for further development as well.

**Reply by endertable** [reply]
> openipc will probably be the project to keep an eye out for T31 devices. I donated a V3 to them.
> 
> They already have some wired T31 devices working, hopefully they can get the V3 working as well. Check out their website [www.openipc.org](http://www.openipc.org)
> 
> They still need a V2, PanV2 and Outdoor V2 for further development as well.

This looks very interesting.  Are any of the T 31 images they have in “Done” status applicable to the V3.   What will be the main advantage of that FW vs. what this project provides, other than, I guess, being able to be completely off of the Wyze infrastructure.  Will it allow web streaming?
